man/docker-create.1.md

```diff
@@ -267,6 +268,10 @@ This value should always larger than **-m**, so you should always use this with
 **-t**, **--tty**=*true*|*false*
    Allocate a pseudo-TTY. The default is *false*.
 
+**--tmpfs**=[] Create a tmpfs mount
+
+   Mount a temporary filesystem (tmpfs) mount within a container. For example,  `--tmpfs /tmp:rw,size=787448k,mode=1777` mounts a tmpfs at /tmp within the container. The mount copies the underlying content into the tmpfs). The supported options are the same as the Linux default mount flags. If you do not specify any options, we default to the following options: `rw,noexec,nosuid,nodev,size=65536k`.
```

Comment:
@rhatdan  Thanks.... `The mount copies` is kind of ambiguos...if you add an example it is clearer.  Also some nits on tics.

```markdown
Mount a temporary filesystem (`tmpfs`) mount into a container, for example:

$ docker run -d --tmpfs /tmp:rw,size=787448k,mode=1777 my_image

This command mounts a `tmpfs` at `/tmp` within the container. The mount copies
the underlying content of `my_image` into `/tmp`. The supported mount options
are the same as the Linux default `mount` flags. If you do not specify any
options, the systems uses the following options:
`rw,noexec,nosuid,nodev,size=65536k`.
```